spec/models/location_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+require 'mongoid_helper'
+
+RSpec.describe Location, type: :model do
+  it { is_expected.to validate_presence_of(:city) }
+  it { is_expected.to validate_presence_of(:region) }
+  it { is_expected.to validate_presence_of(:longlat) }
+  it { is_expected.to have_many(:attractions) }
```

Comment:
I would add a test for the `latitude` and `longitude` methods here.  It seems trival but, as you know, it's easy to get confused when you're just dealing with multi-dimensional arrays. :confused: